packages/gatsby-theme-docs/src/design-system.js

```diff
@@ -39,6 +39,9 @@ export const colors = {
     surfaceCodeHighlight: uikitColors.colorAccent40,
     surfaceInfo: uikitColors.colorInfo95,
     surfaceQuote: uikitColors.colorAccent98,
+    surfaceInfo2: '#eef4f7',
```

Comment:
Are you sure this is correct? It should be `colorInfo95` (also for consistency with how the other colors are picked), which is already mapped as `surfaceInfo`